geomesa-ui/src/main/ui/bower.json

```diff
@@ -14,6 +14,7 @@
     "lodash": "~2.4.1",
     "moment": "~2.7.0",
     "moment-timezone": "~0.1.0",
-    "openlayers" : "https://github.com/openlayers/ol3/releases/download/v3.2.1/v3.2.1.zip"
+    "openlayers" : "https://github.com/openlayers/ol3/releases/download/v3.2.1/v3.2.1.zip",
+    "openlayers2": "https://github.com/openlayers/openlayers/releases/download/release-2.13.1/OpenLayers-2.13.1.zip"
```

Comment:
We haven't ran into any issues yet. We are using them independently. We aren't sending any openlayers2 objects into openlayers3 or vice versa right now. Unless they both try to use the same global variable or something we should be ok.